package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "coderscamp2020.project.typescript.fitnotfat",
+  "version": "1.0.0",
+  "description": "FitNotFat Application - CodersCamp 2021 Typescript project",
+  "main": "index.html",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "format": "prettier --write \"Src/**/*.js\" \"Src/**/*.ts\" \"test/**/*.js\" \"test/**/*.ts\"",
+    "start:dev": "parcel -p 8765 watch index.html",
+    "compile": "tsc",
+    "build": "parcel build index.html --public-url ./"
+
+  },
+  "keywords": [
+    "fitnotfat",
+    "diet",
+    "typescript"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "regenerator-runtime": "^0.13.7"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.20",
+    "jest": "^26.6.3",
```

Comment:
I suggest to match the major and minor version as closely as possible (https://jestjs.io/docs/en/getting-started#using-typescript)

Reply:
done